pygmt/datasets/earth_relief.py

```diff
@@ -11,7 +11,13 @@
 
 
 @kwargs_to_strings(region="sequence")
-def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm=False):
+def load_earth_relief(
+    resolution="01d",
+    region=None,
+    registration=None,
+    use_srtm=False,
+    data_source="relief",
+):
     r"""
     Load Earth relief grids (topography and bathymetry) in various resolutions.
```

Comment:
The bottom lines from L24 to L35 need to be revised as they refer to `earth_relief`, but it should also mention `earth_gebco` now, or be general enough to mention relief/gebco.